---
title: "HPAI outbreak dynamics in North America: 2021-present"
authors: "Anna Jaeger"
affiliations: "Moncla Lab, University of Pennsylvania"
date: "December 2023"
dataset: "https://nextstrain.org/groups/moncla-lab/h5nx/north-america/ha"
abstract: "This report presents an overview of the current outbreak of highly pathogenic avian influenza in North America and highlights the utility of Nextstrain for interrogating outbreak dynamics."
---
# [Introduction to Highly Pathogenic Avian Influenza](https://nextstrain.org/flu/avian/h5n1/ha?animate=1996-03-10,2023-10-11,1,1,30000&d=map&p=full) 

Highly pathogenic avian influenza (HPAI) has disseminated globally since the emergence of HPAI H5N1 in Guangdong China in 1996, resulting in outbreaks primarily throughout Asia, Europe, and Africa. 

**need to figure out how to make the map build go all the way**

Late 2021, HPAI H5N1 was introduced into North America and has rapidly spread throughout the continent. 

Using Nextstrain, we can infer the introductions of this virus into the US, identify patterns of spread between species and regions, and identify mutations that have arisen during this outbreak.


# [Origins and spread of the North American outbreak](https://nextstrain.org/groups/moncla-lab/h5nx/north-america/ha?c=flyway&d=tree,map&p=grid&r=division)

### Origins
Phylogenetic reconstruction allows us to infer three introductions into North America that seeded this outbreak. The major introduction responsible for the majority of infections (the main branch) came from Europe. There were additionally two separate introductions from Asia. ** need to upload global build for this slide ** 

# [Origins and spread of the North American outbreak](https://nextstrain.org/groups/moncla-lab/h5nx/north-america/ha?c=flyway&d=tree,map&p=grid&r=division)

### Origins
Phylogenetic reconstruction allows us to infer three introductions into North America that seeded this outbreak. The major introduction responsible for the majority of infections (the main branch) came from Europe. There were additionally two separate introductions from Asia. 

### Onward spread
When we color by migratory flyway, we can see that the main introduction from Europe was introduced to the Atlantic flyway, whereas the Asian introductions entered via the Pacific flyway. Viruses largely cluster within flyways with some transmission between flyways. Spread into Central America occurs within the Central Flyway. 

# [What is unique about this outbreak?](https://nextstrain.org/groups/moncla-lab/h5nx/north-america/ha?c=Domestic_Status&p=grid&r=division)

There are several unique attributes to this outbreak that have not been recognized in prior outbreaks.

### Unique feature 1: the role of wild birds
The involvement and infection of non-canonical wild bird species and a high rate of non-human mammal infections. In this Nextstrain view, the tree is colored by host type. Here we see a high involvement of wildbirds, as well repeated infections of non-human mammals throughout the phylogeny, and represented throughout the US. 

# [What is unique about this outbreak?](https://nextstrain.org/groups/moncla-lab/h5nx/north-america/ha?c=order&d=tree&p=full&r=region)

There are several unique attributes to this outbreak that have not been recognized in prior outbreaks.

### Unique feature 1: the role of wild birds

This outbreak has seen unprecedented involvement and infection of non-canonical wild bird species and a high rate of non-human mammal infections. The typical main players in H5N1 outbreaks and spread are Anseriformes and Galliformes. If we color the phylogeny by Order, we see that while these are still predominantly featured in this outbreak, there is a large proportion of other avian orders including over 100 acciptriformes infected (eagles, vultures, and hawks), and dozens of strigiformes (owls).  

# [What is unique about this outbreak?](https://nextstrain.org/groups/moncla-lab/h5nx/north-america/ha?c=Domestic_Status&d=tree,map&p=full&r=division)

There are several unique attributes to this outbreak that have not been recognized in prior outbreaks.

### Unique feature 2: high incidences of mammal infections 

In addition to non-canonical bird infections, this outbreak has also resulted in high incidences of mammal infections. These mammal infections have occured throughout North America, and primarily are affecting marine mammals (in the US, harbor seals) and small carnivores (foxes, skunks, etc.)

# [Viral evolution](https://nextstrain.org/groups/moncla-lab/h5nx/north-america/pb2?branches=hide&c=gt-PB2_627&d=tree&gmax=2283&gmin=4&l=scatter&p=full&r=division&scatterX=Domestic_Status&scatterY=gt)

Using Nextstrain filtering functionalities, we can also assess the prevalence of mutations of interest. 

### Mammalian adaptations:

Given the high incidences of mammalian infections during this outbreak, the presence of mammalian host adaptations arising One such mutation is PB2 E267K which is associated with mammalian host adaption. PB2 E267K- as well as a different amino acid substitution in teh same position, E627V- are present in 14% of nonhuman mammal infections. PB2 E267K is also present in wild birds, domestic birds, and a single sampled backyard bird infection. 


# [Viral evolution](https://nextstrain.org/groups/moncla-lab/h5nx/north-america/pb2?c=Domestic_Status&d=tree,map&gmax=2283&gmin=4&gt=PB2.627K&p=grid&r=division)

Using Nextstrain filtering functionalities, we can also identify the prevalence of mutations of interest. 

### Mammalian adaptations:

One such mutation is PB2 E267K which is associated with mammalian host adaption. PB2 E267K- as well as a different amino acid substitution in the same position, E627V- are present in 14% of nonhuman mammal infections. PB2 E267K is also present in wild birds, domestic birds, and a single sapmpled backyard bird infection. 

This view filters for samples containing the PB2 267K mutation, colored by domestic status (Wild Bird, Domestic Bird, Backyard Bird, or Nonhuman Mammal). The muatation is present throughout the phylogeny and in samples from 10 states and one Canadian province, indiciating that this mutation has arisen independently multiple times during this outbreak. 

# [Assessing reassortment](https://nextstrain.org/groups/moncla-lab/h5nx/north-america/ha:groups/moncla-lab/h5nx/north-america/na)

We can also preliminarily assess reassortment of these virsues by visualizing tanglegrams, which pair trees from different gene segments and matches corresponding branches. **something about these not being like very quantitative or accurate without further tree manipulating it gives us a sense that there is likely a lot of reassortment going on in this outbreak**
